vite.config.ts

```diff
@@ -31,8 +21,9 @@ export default ({ command, mode }) => {
       iteriaLowcode({
         command,
         mode,
-        graphQLEndpoint: process.env.VITE_HASURA_GRAPHQL_ENDPOINT,
+        graphQLEndpoint: "https://demo-orders.hasura.app/v1/graphql",
```

Comment:
Let's avoid hardcoded thing. This parameter can be to override the default behaviour. Default behaviour of the plugin should find `VITE_GRAPHQL_ENDPOINT` or `VITE_HASURA_GRAPHQL_ENDPOINT` if it is not overriden here.